packages/cli/src/utils/scaffold.ts

```diff
@@ -67,7 +67,7 @@ export async function buildScaffold(
   };
 
   await execa(
-    'npx',
+    'yarn',
```

Comment:
I think this won't work on the user side, if they don't have yarn. We should stay with `npx` here.